### PR TITLE
Add VCR.py to third party packages doc.

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

**Comment by StephenBrown2** [comment]
Does VCR.py support `httpx`? It's not listed in https://vcrpy.readthedocs.io/en/latest/installation.html#compatibility (yet)

**Comment by jairhenrique** [comment]
@StephenBrown2 supports! https://github.com/kevin1024/vcrpy/releases/tag/v4.1.0

**Comment by StephenBrown2** [comment]
Nice! Looks like the docs just haven't been bumped yet, but it will be there: https://github.com/kevin1024/vcrpy/commit/641d9e5d49c43fe86634c3d99002617ac5330781

File: docs/third-party-packages.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ The ultimate Python library in building OAuth and OpenID Connect clients and ser
 
 An asynchronous GitHub API library. Includes [HTTPX support](https://gidgethub.readthedocs.io/en/latest/httpx.html).
 
+### VCR.py
```
Comment (jcugat) [inline docs/third-party-packages.md:21]: Awesome! Small nitpick though, can you put it under `RESPX` to keep alphabetical order?

Comment (jairhenrique) [reply docs/third-party-packages.md]: @jcugat done!

**Review by jcugat — APPROVED** [review]
👏

**Pull request merged**